server/accesslog.go

```diff
@@ -50,6 +51,10 @@ func (a *AccessLogServer) Register(s *grpc.Server, handler *Handler, d time.Dura
 	a.handler = handler
 	a.streamTimeout = d
 	a.context = ctx
+	a.gatewaySource = defaultGatewaySource
+	if a.handler.operationConfigType == "PROXY" {
```

Comment:
I believe this should use the `product.ProxyOperationConfigType` constant, should it not?

Reply:
Done.